api/net/Areas/Editor/Controllers/AVOverviewController.cs

```diff
@@ -164,16 +164,16 @@ public IActionResult Delete(AVOverviewInstanceModel model)
     /// </summary>
     /// <param name="instanceId"></param>
     /// <returns></returns>
-    [HttpPost("{instanceId}/preview")]
+    [HttpPost("{instanceId}/view")]
```

Comment:
View is a better way to describe an instance.  Preview is before an instance is created.